packages/edge-login-ui-rn/src/native/components/exportAPIComponents/ChangePinScreen.js

```diff
@@ -32,23 +33,31 @@ class ChangePinScreen extends Component<Props> {
   }
   store: Store<State, Action>
   componentWillMount () {
-    setLocal(
-      this.props.locale || ChangePinScreen.defaultProps.locale,
+    // Why are these optional for this screen and not for the others!?:
+    const locale = this.props.locale || ChangePinScreen.defaultProps.locale
+    const language =
       this.props.language || ChangePinScreen.defaultProps.language
-    )
+    setLocal(locale, language)
```

Comment:
The PIN and Recovery scenes both allow `locale` and `language` to be optional, but the other scenes don't for some reason. We should probably be consistent.

Reply:
They don't allow it to be optional, they set it to a default if for some reason it is not passed in by the parent app.